release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1638,6 +1677,8 @@ In this update, `systemd` service only sets the default RPS mask for virtual int
 
 * This fix ensures that read-only-many volumes are provisioned appropriately by GCP CSI Driver as read-only. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1968253#[*BZ#1968253*])
 
+* {product-title} allows installation of of vSphere CSI drivers shipped by either the upstream community or VMware. Although Red Hat does not support this driver, cluster administrators ca still install and use it because it has more features than the vSphere CSI driver shipped by Rd Hat. {product-title} can be upgraded to {product-version} with the upstream and VMware vSphere CSI driver, however, customers will be warned about the presence of a third party CSI driver. In a future release {product-title}, customers will be required to upgrade to the version of vSphere CSI shipped by Red Hat before upgrading to the next version of {product-title}. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=2089419[BZ#2089419] and link:https://bugzilla.redhat.com/show_bug.cgi?id=2052071[*BZ#2052071*]).
```

Comment:
```suggestion
* {product-title} allows installation of of vSphere CSI drivers shipped by either the upstream community or VMware. Although Red Hat does not support this driver, cluster administrators can still install and use it because it has more features than the vSphere CSI driver shipped by Red Hat. {product-title} can be upgraded to {product-version} with the upstream and VMware vSphere CSI driver, however, you will be warned about the presence of a third party CSI driver. In a future release {product-title}, you will be required to upgrade to the version of vSphere CSI shipped by Red Hat before upgrading to the next version of {product-title}. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=2089419[BZ#2089419] and link:https://bugzilla.redhat.com/show_bug.cgi?id=2052071[*BZ#2052071*]).
```